src/utils/tracked-controls.js

```diff
@@ -48,7 +49,7 @@ module.exports.isControllerPresent = function (sceneEl, idPrefix, queryObject) {
     isPrefixMatch = (!idPrefix || idPrefix === '' || gamepad.id.indexOf(idPrefix) === 0);
     isPresent = isPrefixMatch;
     if (isPresent && queryObject.hand) {
-      isPresent = gamepad.hand === queryObject.hand;
+      isPresent = (gamepad.hand || DEFAULT_HANDEDNESS) === queryObject.hand;
```

Comment:
if queryObject.hand === 'left' and no gamepad.hand, DEFAULT_HANDEDNESS should be used so isPresent becomes false, right?  with yours, isPresent stays true